albumentations/core/composition.py

```diff
@@ -210,6 +210,22 @@ def _to_dict(self):
         )
         return dictionary
 
+    def get_dict_with_id(self):
+        dictionary = super(Compose, self).get_dict_with_id()
```

Comment:
Since Albumentations is Python 3 only now, I think it is better to use a simplified syntax for calling superclass: `super().get_dict_with_id()`